### PR TITLE
add mongo workflow

### PR DESCRIPTION
**Pull request by Uzlopak**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by Uzlopak** [comment]
Should we use mongo 6? Should we add parameters?

File: .github/workflows/plugins-ci-mongo.yml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        required: false
+        default: 'fastify'
+        type: string
+      license-check:
+        description: 'Check licenses'
+        required: false
+        type: boolean
+        default: false
+      license-check-allowed-additional:
+        description: 'Additional allowed licenses (separator ;)'
```
Comment (Fdawgs) [inline .github/workflows/plugins-ci-mongo.yml:17]:
```suggestion
        description: 'A semicolon seperate list of additional licenses to allow.'
```

**Comment by climba03003** [comment]
> Should we use mongo 6? Should we add parameters?

I would check both version. 
But from my experience, server version shouldn't makes different since we check the connection only in `@fastify/mongodb`.

**Comment by Uzlopak** [comment]
There is no official mongo:6.0.0 docker image it seems like.

File: .github/workflows/plugins-ci-mongo.yml [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  test:
+    name: Node.js ${{ matrix.node-version }} - ${{ matrix.db }}
+    runs-on: ${{ matrix.os }}
+    permissions:
+      contents: read
+    strategy:
+      matrix:
+        node-version: [14, 16, 18]
+        os: [ubuntu-latest]
+        db: ['mongo:5.0.0']
```
Comment (Fdawgs) [inline .github/workflows/plugins-ci-mongo.yml]: From looking at https://hub.docker.com/_/mongo/tags, maybe just use 5, 5.0, or 5.0.10?

Comment (Uzlopak) [reply .github/workflows/plugins-ci-mongo.yml]: done

**Review by Fdawgs — APPROVED** [review]

**Pull request merged**